grafana-plugin/src/containers/IntegrationForm/IntegrationForm.tsx

```diff
@@ -414,11 +419,19 @@ export const IntegrationForm = observer(
         labels: labels ? [...labels] : undefined,
       };
 
-      if (formData.integration !== 'servicenow') {
+      const isServiceNow = formData.integration === 'servicenow';
+      const isCreate = id === 'new';
```

Comment:
`nit` I think it's better to create vars if they are consumed more than once only or if they contans complex logic behind (a large set of consecutive OR and AND statements), in this case adding a `isCreate` var looks redundant

Reply:
IMO this healps with readability, much easier to read `isCreate` than `id==='new'`, we have some other places like this as well

`isNew` for example on IntegrationForm

Reply:
Done, didn't notice we already had `isNew`